src/components/Calculator.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Result from './Result';
+import Buttons from './Buttons';
+
```

Comment:
- [ ] The Calculator component should import the calculate.js file from the logic directory, please do so and refactor your code as necessary to comply with the project requirements.